README.md

```diff
@@ -1,37 +1,104 @@
-## Welcome to GitHub Pages
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-You can use the [editor on GitHub](https://github.com/ijeomaodinko/CAPSTONE_2/edit/develop/README.md) to maintain and preview the content for your website in Markdown files.
+# Project Name
 
-Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.
+This project contains codes that enables users get from and add data to the API. It shows information about movie shows. People can browse through, like and comment. The count of likes and count are registed.
 
-### Markdown
 
-Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for
+## Built With
 
-```markdown
-Syntax highlighted code block
+- HTML
+- CSS
+- Javascript
+- VsCode
+- Webpack
 
-# Header 1
-## Header 2
-### Header 3
+## Live Demo (if available)
 
-- Bulleted
-- List
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] I think you forget to deploy your project so kindly make sure to deploy it since it is one of the requirements ✅✅🙏🏻🙏🏻